backend/src/Edna/Library/DB/Query.hs

```diff
@@ -178,30 +178,49 @@ getMethodologyByName name = runSelectReturningOne' $ select $ do
   pure methodologies
 
 -- | Get all methodologies
-getMethodologies :: Edna [TestMethodologyRec]
-getMethodologies = runSelectReturningList' $ select $ all_ $ esTestMethodology ednaSchema
+getMethodologies :: Edna [(TestMethodologyRec, [Text])]
+getMethodologies = getMethodology' Nothing
+
+getMethodology' :: Maybe MethodologyId -> Edna [(TestMethodologyRec, [Text])]
+getMethodology' mMethodologyId =
+  fmap convert $
+  runSelectReturningList' $ select $
+  pgNubBy_ (bimap tmTestMethodologyId pProjectId) $ do
```

Comment:
I think we also need `orderBy_`

Reply:
No, because I use `NE.groupAllWith` which doesn't require sorting.